crates/adapters/src/format/parquet/test.rs

```diff
@@ -163,9 +163,23 @@ fn parquet_output() {
         .write(&batch)
         .expect("Writing to parquet should succeed");
     writer.close().expect("Closing the writer should succeed");
-    debug_parquet_buffer(buffer.lock().unwrap().concat());
+    debug_parquet_buffer(
```

Comment:
does this do lazy evaluation of the argument?

Reply:
this is inefficient, but this is test code.